### DISCUSSION TITLE
Internet

### DISCUSSION THREAD

**Discussion by osdevk** [CosmosOS/Cosmos#1700] [Q&A]
Is it possible to use the internet on Cosmos yet, or no?

**Comment by quajak** [comment]
There are drivers and implementations for some of the fundamental protocols, so it is possible to do basic networking. What are you trying to do?

**Reply by osdevk** [reply]
Simple stuff like e-mails

**Reply by quajak** [reply]
@valentinbreiz knows more than I about the current state of networking but AFAIK no work has been done on implementing the protocols required for networking working.

**Reply by ghost** [reply]
You make the smtp drivers yourself :trollface:
